From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into peering-tutorials

davidmu1 committed:
freshness update

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into peering-tutorials

TerryLanfear committed:
fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sec-paas

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into peering-tutorials

billmath committed:
updating freshness

dsk-2015 committed:
CELA approved notes added to all hub and device ID includes

dsk-2015 committed:
resolved merge conflict in one of the includes

Commit:
e and Dv3

MarkMcGeeAtAquent committed:
Merge pull request #2 from MicrosoftDocs/master

Synce atttempt

dsk-2015 committed:
alignment and java note

Juliako committed:
Update media-services-dotnet-live-encode-with-onpremises-encoders.md

Juliako committed:
Update media-services-dotnet-manage-entities.md

dsk-2015 committed:
java url fix

TerryLanfear committed:
new feature

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into security-jit

Commit:
Merge pull request #17515 from Juliako/patch-3

Update media-services-dotnet-live-encode-with-onpremises-encoders.md

Commit:
Merge pull request #17516 from Juliako/patch-4

Update media-services-dotnet-manage-entities.md

v-thepet committed:
Merge pull request #17126 from v-nagta/five9

USER STORY 1035570 SaaS App Tutorial: Five9 Plus Adapter (CTI, Contac…

Commit:
Merge pull request #17494 from gbowerman/patch-7

Add PowerShell examples for querying public IPs

MicrosoftGuyJFlo committed:
Retiring content for LinkedIn Lookup per vendor request

Commit:
update storage doc again

TerryLanfear committed:
TOC entry

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into security-jit

Commit:
Merge pull request #17513 from cynthn/e-acu

E and Dv3 ACUs

Commit:
Merge pull request #17519 from MicrosoftGuyJFlo/RetireLinkedINLookup

Retiring content for LinkedIn Lookup per vendor request

ErikjeMS committed:
updated password exp reset process

Juliako committed:
Update media-services-compare-encoders.md

Juliako committed:
Update media-services-community.md

Commit:
adding two relnotes

MarkMcGeeAtAquent committed:
Cosmos DB: final link fixes

MicrosoftGuyJFlo committed:
Retiring content for LinkedIn Lookup per vendor request

Juliako committed:
Update media-services-develop-video-players.md

Commit:
Merge pull request #17521 from Juliako/patch-4

Update media-services-compare-encoders.md

Commit:
Merge pull request #17522 from Juliako/patch-5

Update media-services-community.md

Commit:
Merge pull request #17526 from Juliako/patch-6

Update media-services-develop-video-players.md

Commit:
Merge pull request #17525 from ErikjeMS/master

updated password exp reset process

MarkMcGeeAtAquent committed:
Cosmos DB: link fix

Commit:
Merge pull request #17527 from MarkMcGeeAtAquent/AfterSync

Cosmos DB: link fix

ErikjeMS committed:
updates per Heath

ErikjeMS committed:
Merge pull request #1 from HeathL17/patch-4

Update azure-stack-provision-vm.md

dsk-2015 committed:
Adding to other iot hub create files and fixing alignment and numbering

dsk-2015 committed:
hub note url fix

rwike77 committed:
fixed broken link

rwike77 committed:
fixed broken link

Commit:
Merge pull request #17523 from HeathL17/hl-relnotes

adding two relnotes

dsk-2015 committed:
alignment fix

v-thepet committed:
Merge pull request #17517 from TerryLanfear/security-jit

Security jit

v-thepet committed:
Merge pull request #17365 from jluk/jluk-cloud-shell

add advanced storage content

v-thepet committed:
Merge pull request #17524 from MicrosoftGuyJFlo/RedirectAndDeleteLinkedInLookup

Retiring content for LinkedIn Lookup per vendor request

v-thepet committed:
Merge pull request #17531 from rwike77/tocfix

fixed broken link

v-thepet committed:
third round of broken links today

Commit:
Merge pull request #17533 from v-thepet/today

production bug: third round of broken links today

charwen committed:
Uploaded the picture to fix a typo

Commit:
Merge pull request #17520 from zivraf/LiveMigrationDocfix

fix typo

charwen committed:
Uploaded the picture to fix a typo

dsk-2015 committed:
alignment fix2

Commit:
Merge pull request #17504 from billmath/aadconnect2

updating freshness

Commit:
quickfix01

Commit:
quickfix02

Commit:
quickfix03

Commit:
Merge pull request #17535 from MarkusVi/quickfix01

quickfix01

Commit:
quickfix04

Commit:
quickfix05

Commit:
Merge pull request #17536 from MarkusVi/quickfix02

quickfix02

Commit:
Merge pull request #17537 from MarkusVi/quickfix03

quickfix03

Commit:
Merge pull request #17538 from MarkusVi/quickfix04

quickfix04

Commit:
Merge pull request #17539 from MarkusVi/quickfix05

quickfix05

GitHubber17 committed:
Merge pull request #17406 from charwen/master

Updated the ExpressRoute architecture picture

genlin committed:
update the download link

rboucher committed:
delete one file and redirect

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into addquickstart

rboucher committed:
correct to Quickstart from QuickStart

Commit:
quickfix01

Commit:
Merge pull request #17540 from genlin/64313

How-to-use-perfInsights.md

Commit:
Merge pull request #17542 from MarkusVi/quickfix01

quickfix01

Commit:
redir01

GitHubber17 committed:
Merge pull request #17189 from rboucher/fixmetadata

fixing all author and manager metadata

Juliako committed:
Update media-services-dotnet-live-encode-with-onpremises-encoders.md

Juliako committed:
Update media-services-dotnet-live-encode-with-onpremises-encoders.md

Juliako committed:
Update media-services-dotnet-creating-live-encoder-enabled-channel.md

Juliako committed:
Update media-services-dotnet-creating-live-encoder-enabled-channel.md

Juliako committed:
Update hybrid-design-drm-sybsystem.md

TerryLanfear committed:
final fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sec-paas

Commit:
Merge pull request #17546 from Juliako/patch-7

Update hybrid-design-drm-sybsystem.md

TerryLanfear committed:
acrolinx fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sec-paas

Commit:
Merge pull request #17505 from TerryLanfear/sec-paas

Sec paas

shruja committed:
update

shruja committed:
update

JasonWHowell committed:
Update connect-nodejs.md

Fixing some typos.

shruja committed:
update

shruja committed:
update

Commit:
Merge pull request #17548 from JasonWHowell/patch-2

MySQL: Update connect-nodejs.md

genlin committed:
apply editing changes

genlin committed:
correct typo

shruja committed:
update

genlin committed:
fix link issue

Commit:
quickfix01

Commit:
Merge pull request #17555 from MarkusVi/quickfix01

quickfix01

v-nagta committed:
updated

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into peering-tutorials

vhorne committed:
replace wrong characters

Commit:
Fixes.

Commit:
Merge pull request #17559 from vhorne/char-fix

replace wrong characters

Commit:
Added note, expanded NIC acronym

Commit:
Fix.

Commit:
Fixed.

Commit:
Fixes.

Commit:
Fixes.

matt1883 committed:
[ADLA] .NET SDK List pipelines and recurrences

matt1883 committed:
Update data-lake-analytics-manage-use-dotnet-sdk.md

Commit:
Merge pull request #17562 from matt1883/patch-1

[ADLA] .NET SDK List pipelines and recurrences

matt1883 committed:
[ADLA] Python - Recurrences, policies

Added content to ADLA manage Python article to cover listing pipelines, listing recurrences, listing compute policies, and creating compute policies.

matt1883 committed:
Update data-lake-analytics-manage-use-python-sdk.md

Commit:
Typo fix

Commit:
Typo fix

Commit:
Typo fix

Commit:
Update data-lake-analytics-manage-use-powershell.md

Commit:
parallel - asynchronous

rjagiewich committed:
Merge pull request #17563 from matt1883/patch-2

[ADLA] Python - Recurrences, policies

rjagiewich committed:
Merge pull request #17561 from jimdial/master

Fixed.

rjagiewich committed:
Merge pull request #17560 from jimdial/patch-92

Added note, expanded NIC acronym

msmbaldwin committed:
Update TOC.md

rjagiewich committed:
Merge pull request #17551 from genlin/64148

virtual-network-troubleshoot-cannot-delete-vnet.md

Commit:
quickfix02

Lisaco88 committed:
Merge pull request #17067 from msmbaldwin/ara

First draft

Lisaco88 committed:
Merge pull request #17054 from rboucher/addquickstart

add quickstart link to TOC

Commit:
Fix

Commit:
Merge pull request #17567 from DeniseMak/faq-bugfix-7-14

Faq bugfix 7 14

Commit:
Merge pull request #17565 from saveenr/patch-5

Update data-lake-analytics-manage-use-powershell.md

Commit:
Merge pull request #17564 from HeathL17/hl-deploy-apps

Typo fix

Lisaco88 committed:
Merge pull request #17498 from davidmu1/commonupdate071717

freshness update

Lisaco88 committed:
Merge pull request #15905 from v-viinde/iqnavigatorvms

USER STORY 1023149 SaaS App Tutorial: IQNavigator VMS

Commit:
Merge pull request #17570 from MarkusVi/quickfix02

quickfix02

rjagiewich committed:
Merge pull request #17549 from psignoret/patch-11

Minor code formatting update

Lisaco88 committed:
Merge pull request #17361 from jimdial/peering-tutorials

New scenarios.

Commit:
quickfix03

Lisaco88 committed:
Merge pull request #17028 from v-viinde/liquidfiles

USER STORY 1034730 SaaS App Tutorial: LiquidFiles Tutorial Update

rjagiewich committed:
Merge pull request #17545 from Juliako/patch-5

Update media-services-dotnet-creating-live-encoder-enabled-channel.md

rjagiewich committed:
Merge pull request #17544 from Juliako/patch-4

Update media-services-dotnet-live-encode-with-onpremises-encoders.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Merge pull request #17571 from MarkusVi/quickfix03

quickfix03

ggailey777 committed:
Removed extraneous publish step

kgremban committed:
Remove public preview

vhorne committed:
update for ASDK

v-alje committed:
Merge pull request #17572 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Update data-lake-analytics-manage-use-powershell.md

Commit:
quickfix05

Commit:
Merge pull request #17574 from kgremban/jul18_removepp

Remove public preview

ErikjeMS committed:
Update azure-stack-provision-vm.md

ErikjeMS committed:
blur id

Commit:
Merge pull request #17579 from MarkusVi/quickfix05

quickfix05

Commit:
quickfix06

Commit:
Merge pull request #17580 from MarkusVi/quickfix06

quickfix06

Commit:
quickfix07

Commit:
Updated fundamentals and includes.

jeffgilb committed:
Merge branch 'master' into jeff718

v-thepet committed:
Merge pull request #17576 from ggailey777/master

Removed extraneous publish step

Commit:
Merge pull request #17581 from MarkusVi/quickfix07

quickfix07

v-thepet committed:
Merge pull request #17582 from jeffgilb/jeff718

Updated fundamentals and includes.

v-thepet committed:
Merge pull request #17575 from vhorne/upd-viewpublicip

update for ASDK

v-thepet committed:
Merge pull request #17529 from ErikjeMS/master

updates per heath

Commit:
quickfix08

billmath committed:
updating

Commit:
redir01

Commit:
Merge pull request #17583 from MarkusVi/quickfix08

quickfix08

Commit:
Merge pull request #17584 from billmath/aadconnect2

updating

billmath committed:
hybrid update

Commit:
quickfix09

v-thepet committed:
Merge pull request #17543 from MarkusVi/redir01

redir01

Commit:
Merge pull request #17585 from billmath/aadconnect2

hybrid update

gsilva5 committed:
Changing the warning from !include to !warning embedded in page

Commit:
Merge pull request #17588 from gsilva5/patch-6

Changing the warning from !include to !warning embedded in page

v-thepet committed:
Merge pull request #17123 from shruja/jiramicrosoft

USER STORY 1035552 : SaaS App Tutorial: JIRA SAML SSO by Microsoft

v-thepet committed:
Merge pull request #17125 from shruja/confluencemicrosoft

USER STORY 1035554: SaaS App Tutorial: Confluence SAML SSO by Microsoft

Commit:
Merge pull request #17586 from MarkusVi/quickfix09

quickfix09

Commit:
quickfix10

Juliako committed:
Update media-services-dotnet-live-encode-with-onpremises-encoders.md

rboucher committed:
fix error. missing comma

Juliako committed:
Update media-services-dotnet-manage-streaming-endpoints.md

Commit:
redir02

YuriDio committed:
small update

Commit:
Merge pull request #17592 from Juliako/patch-4

Update media-services-dotnet-live-encode-with-onpremises-encoders.md

neilpeterson committed:
updated cloud shell

Commit:
Merge pull request #17589 from MarkusVi/quickfix10

quickfix10

Commit:
Update data-lake-analytics-manage-use-powershell.md

Juliako committed:
Update media-services-protect-hls-with-fairplay.md

neilpeterson committed:
path issue

Commit:
Merge pull request #17595 from YuriDio/JulyUpdates_3

small update

v-thepet committed:
Merge pull request #17594 from MarkusVi/redir02

redir02

Commit:
Merge pull request #17596 from neilpeterson/acs-quickstart-cloudshell

updated cloud shell

Juliako committed:
Update media-services-protect-hls-with-fairplay.md

v-thepet committed:
Merge pull request #17136 from v-nagta/ceridianupdate

USER STORY 1035616 SaaS App Tutorial: Ceridian Dayforce HCM Update

rboucher committed:
fixing merge conflict

Juliako committed:
Update media-services-protect-with-aes128.md

Juliako committed:
Update media-services-protect-with-aes128.md

anzaman committed:
Update expressroute-locations.md

CarlRabeler committed:
Update sql-database-design-first-database.md

v-thepet committed:
Merge pull request #17578 from saveenr/patch-6

Update data-lake-analytics-manage-use-powershell.md

Juliako committed:
Update media-services-protect-with-drm.md

Juliako committed:
Update media-services-protect-hls-with-fairplay.md

Commit:
Merge pull request #17599 from anzaman/patch-8

Update expressroute-locations.md

Commit:
Merge pull request #17600 from CarlRabeler/patch-301

Update sql-database-design-first-database.md

Commit:
Update aas-lesson-13-deploy.md

v-thepet committed:
Merge pull request #17593 from Juliako/patch-8

Update media-services-dotnet-manage-streaming-endpoints.md

Commit:
redir03

v-thepet committed:
Merge pull request #17597 from Juliako/patch-9

Update media-services-protect-hls-with-fairplay.md

Commit:
Update aas-lesson-13-deploy.md

Commit:
Update aas-lesson-13-deploy.md

Ja-Dunn committed:
Merge pull request #17604 from MarkusVi/redir03

redir03

Ja-Dunn committed:
Merge pull request #17601 from Juliako/patch-11

Update media-services-protect-with-drm.md

Lisaco88 committed:
Merge pull request #17598 from Juliako/patch-10

Update media-services-protect-with-aes128.md

Commit:
quickfix11

Commit:
redir04

rboucher committed:
spelling error and remove redirected article

Ja-Dunn committed:
Merge pull request #17152 from Santoshtambe/Bug_1035158

Bug 1035158  changes implemented

ahmedelnably committed:
Fixing styling issues

Fixing styling issues

Commit:
Merge pull request #17606 from MarkusVi/quickfix11

quickfix11

Ja-Dunn committed:
Merge pull request #17602 from Minewiskan/patch-12

Update aas-lesson-13-deploy.md

Ja-Dunn committed:
Merge pull request #17607 from MarkusVi/redir04

redir04

Ja-Dunn committed:
Merge pull request #17532 from rwike77/lpfix

fixed broken .NET reference link

rboucher committed:
another merge conflict

Ja-Dunn committed:
Merge pull request #17274 from v-atmapa/master

1035178 - Updated Images added

rboucher committed:
another merge conflict

rboucher committed:
another merge conflict

Commit:
tocupdate01

pharring committed:
Update ASP.NET Core sample code

Commit:
added important note to ensure that customers use the preview version of visual studio

Ja-Dunn committed:
Merge pull request #17299 from dsk-2015/dsk_iot

Adding note about PII

Commit:
Merge pull request #17608 from ahmedelnably/master

Fixing styling issues

adegeo committed:
Removed bad perf articles and references

Ja-Dunn committed:
Merge pull request #17610 from MarkusVi/tocupdate01

tocupdate01

Ja-Dunn committed:
Merge pull request #17541 from rboucher/redirecthowtouse2

delete one file and redirect

adegeo committed:
Updated redirect file

adegeo committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into vso-bad

Commit:
Merge pull request #17611 from SyntaxC4-MSFT/callout-vs-preview

Providing a Callout for VS Preview 15.3 to avoid customer confusion

adegeo committed:
Updated freshness

Commit:
Merge pull request #17613 from pharring/master

Update ASP.NET Core sample code

Ja-Dunn committed:
Merge pull request #17612 from Thraka/vso-bad

Removed bad articles

Ja-Dunn committed:
Merge pull request #17615 from Thraka/freshness

Updated freshness

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #17620 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
